GNUmakefile

```diff
@@ -0,0 +1,29 @@
+
+
+default: build
+
+ifneq ($(PRODUCT),)
+  mmv1_compile=-p products/$(PRODUCT)
+  tpgtools_compile = --service $(PRODUCT)
+else
+  mmv1_compile=-a
+  tpgtools_compile =
+endif
+ifneq ($(RESOURCE),)
+  mmv1_compile += -t $(RESOURCE)
+  tpgtools_compile += --resource $(RESOURCE)
+endif
+build:
+	make mmv1
+	make tpgtools
+
+mmv1:
```

Comment:
Will `make mmv1 OUTPUT_PATH=/path/to/terraform-provider-google VERSION=ga PRODUCT=compute` (and  the same for tpgtools) work / can we make it work? I'd love to eliminate the need for calling the tools directly entirely outside of advanced usecases.

Reply:
Yup that works! As does `make OUTPUT_PATH=/path/to/terraform-provider-google VERSION=ga PRODUCT=compute` which compiles both tpgtools/mmv1 at GA for compute.

We will probably need to add the no-code and no-docs flags to tpgtools at some point to replicate current website docs behavior